aad/Module.md

```diff
@@ -1,91 +1,33 @@
 ## Overview
-Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language 
-easily. It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent 
+Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language. 
+It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent 
```

Comment:
Do we need to use `` for the resource types. And I don't think we need to capitalize them as well.

Reply:
Fixed